app/assets/javascripts/course/discussion/topics.js

```diff
@@ -29,4 +31,7 @@
 
   $(document).on('page:load ready', showCommentBoxes);
```

Comment:
Think we should change all `page:load` to `$(document).on "turbolinks:load"`: http://guides.rubyonrails.org/working_with_javascript_in_rails.html#turbolinks

`page:load` doesn't get called any more from my experiment.